deployment/kubernetes/deployment.yml

```
@@ -0,0 +1,66 @@
+          livenessProbe:
+            httpGet:
+              path: /health
+              port: 8000
+            initialDelaySeconds: 30
+            periodSeconds: 10
+          readinessProbe:
+            httpGet:
+              path: /ready
+              port: 8000
+            initialDelaySeconds: 15
+            periodSeconds: 5
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)